CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.0.12]
+
+### Changed
+
+- Use PricesIncludeTax property on market to determine if tax should be included on orderline
```

Comment:
Should this maybe be mentioned in the documentation as well?

Reply:
This is something that Klarna requires and is built inside the package. The developer doesn't have any control (could override but that will only cause problems) over this.